### PR TITLE
Setting up vbs integration

### PR DESCRIPTION
**Pull request by nyospe**
Resolves https://github.com/EspressoSystems/versioned-binary-serialization/issues/25

### CODE REVIEW

File: src/availability.rs [hunk 1]
```diff
@@ -140,13 +141,16 @@ impl Error {
...
 where
     State: 'static + Send + Sync + ReadState,
     <State as ReadState>::State: Send + Sync + AvailabilityDataSource<Types>,
     Payload<Types>: QueryablePayload,
 {
-    let mut api = load_api::<State, Error>(
+    let mut api = load_api::<State, Error, MAJOR_VERSION, MINOR_VERSION>(
         options.api_path.as_ref(),
         include_str!("../api/availability.toml"),
         options.extensions.clone(),
```
[the remark this answers is not in the document]
Comment (nyospe) [reply src/availability.rs:156]: Good call, but I think, instead, I'll add it as a default behavior in tide-disco and remove the `.with_version` calls on all the implementations. WDYT?

[the remark this answers is not in the document]
Comment (jbearer) [reply src/availability.rs:156]: Yeah I like that

[the remark this answers is not in the document]
Comment (jbearer) [reply src/availability.rs:156]: Hmm. I actually don't necessarily thing the API and executable version should be locked. E.g. performance improvements to the executable, or features outside the scope of the versioned API, like rate limiting, API credits, etc.

Mayyybe we can have (major, minor) locked to the API version and use patch by convention for non-API updates. Or, we can just have separate versions, in which case I think the thing returned by the version endpoint should be the API version. We can make changes on tide disco if necessary to stop using semver and just use major,minor

File: Cargo.toml [hunk 2]
```diff
@@ -100,6 +101,7 @@ native-tls = { version = "0.2", optional = true }
 postgres-native-tls = { version = "0.5", optional = true }
 refinery = { version = "0.8", features = ["tokio-postgres"], optional = true }
 serde_json = { version = "1.0", optional = true }
+tempdir = { version = "0.3", optional = true }
```
Comment (jbearer) [inline Cargo.toml:104]: ?

**Review by jbearer — APPROVED** [review]

File: src/availability.rs [hunk 3]
```diff
@@ -503,7 +508,7 @@ mod test {
 
     /// Get the current ledger height and a list of non-empty leaf/block pairs.
     async fn get_non_empty_blocks(
-        client: &Client<Error>,
+        client: &Client<Error, 0, 1>,
```
[the remark this answers is not in the document]
Comment (nyospe) [reply src/availability.rs:512]: Yeah, I think we can do one better, even... stay tuned, I'm still trying to wrangle all of the dependencies back to passing tests.

File: Cargo.toml (hunk 2 above)
Comment (nyospe) [reply Cargo.toml:104]: No idea where that came in from, I'm removing it...

**Review by jbearer — APPROVED** [review]

**Pull request merged**